### PR TITLE
zh-cn: update the translation of the `Headers.set()` method

### PR DESCRIPTION
**Pull request by yin1999**
### Description

update the translation of the `Headers.set()` method

### CODE REVIEW

File: files/zh-cn/web/api/headers/set/index.md [hunk 1]
```diff
@@ -1,68 +1,70 @@
...
 ---
 
-{{APIRef("Fetch")}}
+{{APIRef("Fetch API")}} {{AvailableInWorkers}}
 
-{{domxref("Headers")}} 接口中 **`set()`** 方法在可以在已经声明中的 **`headers`** 对象修改已有的一组键值对或者创建一个新的键值对。
+{{domxref("Headers")}} 接口的 **`set()`** 方法为 `Headers` 对象中已存在的标头设置新值，或在标头不存在的情况下添加该标头。
 
-**`set()`** 方法和 [append()](/zh-CN/docs/Web/API/Headers/append)方法不同的是声明的 **`Headers`** 对象是否已经存在对应的 **`keys`** 是否已经存在并且已经赋值。**`set()`** 方法将会覆盖之前的 **`value`**，然而 [append()](/zh-CN/docs/Web/API/Headers/append)方法只会在 **`Headers`** 对象的尾部添加一个新的键值对。
+`set()` 和 {{domxref("Headers.append")}} 不同的是如果指定的标头已经存在且其可以接受多个值，`set()` 会使用新值覆盖已存在的值，而 {{domxref("Headers.append")}} 将新值追加到值集的末尾。
```
Comment (yin1999) [inline files/zh-cn/web/api/headers/set/index.md:12]:
```suggestion
`set()` 和 {{domxref("Headers.append")}} 的不同之初在于：如果指定的标头已经存在且其可以接受多个值，`set()` 会使用新值覆盖已存在的值，而 {{domxref("Headers.append")}} 将新值追加到值集的末尾。
```

File: files/zh-cn/web/api/headers/set/index.md [hunk 2]
```diff
@@ -1,68 +1,70 @@
...
 
 ```js
 myHeaders.set("Accept-Encoding", "deflate");
 myHeaders.set("Accept-Encoding", "gzip");
-myHeaders.get("Accept-Encoding"); // Returns 'gzip'
+myHeaders.get("Accept-Encoding"); // 返回“gzip”
 ```
 
-如果你需要增加一个键值对，而不是要覆盖之前的键值对，那么你需要用[append()](/zh-CN/docs/Web/API/Headers/append)方法。
+如果你想要将新值追加到已有的值中（而部署覆盖它），那么你需要使用 {{domxref("Headers.append")}} 方法。
```
Comment (yin1999) [inline files/zh-cn/web/api/headers/set/index.md:56]:
```suggestion
如果你想要将新值追加到已有的值中（而不是覆盖它），那么你需要使用 {{domxref("Headers.append")}} 方法。
```

**Review by JasonLamv-t — COMMENTED** [review]
Just a nit. 👍

**Review by JasonLamv-t — APPROVED** [review]
LGTM.

**Pull request merged**